.github/dependabot.yml

```diff
@@ -8,4 +8,4 @@ updates:
   - package-ecosystem: "pip" # See documentation for possible values
     directory: "/" # Location of package manifests
     schedule:
-      interval: "weekly"
+      interval: "daily"
```

Comment:
Why? Weekly isn't enough?

Reply:
GitHub recommend a daily usage of dependabot to manager a better security :)